### ISSUE TITLE
Failed version check for grimoirelab-panels in 18.07-03

### ISSUE THREAD

**Issue by jgbarah** [chaoss/grimoirelab#128]
Running (from grimoirelab repo):

```
cd docker
ansible-playbook ansible_release.yml --extra-vars "release=18.07-03" --tags "build"
...
Failed version check for grimoirelab-panels
```

**Comment by acs** [comment]
It is grimoirelab-sigils what it is failing, and we are not using master branch in the release, but the migration branch `kibana6-migration`. AFAIK we don't have versions for this branch. So in this case, I would create the pip packages using master version (which has a version), and once this migration branch is in master, the problem will be fixed.

**Comment by jgbarah** [comment]
We need release files that work, and that include that they produce a package with a new version number, if they changed. I still think that the commit from all repos should come from master, but in the end, maybe that doesn't matter that much: the important thing is that the commit in the release file matches a commit that changes the release number in the repo.

That's why I think it is a bug in the release file in this repo.

**Comment by acs** [comment]
Yes, I understand your point, but we are not releasing a new version in panels `kibana6-migration` we are just getting the last state of this migration process. We can create an artificial version in order to have it in the release, but in this case, my guess is that this "release" is just a way to deploy some specific state of the repositories, not real versions. But, what is a version? Every commit in git generates a new "version", something pretty cool in git. But for generating the pip packages, we need an additional version to the git raw version (the commit).

So in this case, we have two options:

* Don't try to generate pip packages for 18.07-03
* Create a pip version for the current status of the `kibana6-migration` sigils branch

What do you prefer @jgbarah ? I am happy with both.

**Comment by jgbarah** [comment]
We need to produce new versions of the pip packages, since it is a number of releases failing now. So, I suggest:

* In the short term, please decide if the commit for the release should be from `master` or from `kibana6-migration`. From whichever you decide it to be, just ensure `_version.py` gets a new number. That's enough to produce new pip packages. I suggest that you just do it with `kibana6-migration`, and that's it.

* In the long term, I'm going to produce a pr for chaoss/grimoirelab-sigils#219 and then, if needed, another one for kidash, if you find that ok.

BTW, 18.07-05 is still failing, but I guess you already know that ;-)

**Comment by acs** [comment]
Fixed all in 18.07-06. Closing this ticket!
